recipes/r-shinyngs/build.sh

```diff
@@ -1,18 +1,15 @@
 #!/bin/bash
 
-# 'Autobrew' is being used by more and more packages these days
-# to grab static libraries from Homebrew bottles. These bottles
-# are fetched via Homebrew's --force-bottle option which grabs
-# a bottle for the build machine which may not be macOS 10.9.
-# Also, we want to use conda packages (and shared libraries) for
-# these 'system' dependencies. See:
-# https://github.com/jeroen/autobrew/issues/3
-export DISABLE_AUTOBREW=1
+# The reason for the devtools install is that uploads to shinyapps.io only
+# works with GitHub dependencies installed via devtools (see
+# https://docs.rstudio.com/shinyapps.io/getting-started.html#important-note-on-github-packages).
+# I'm relying on the PKG hash to maintain consistency with the hash in the
+# meta.yaml.
```

Comment:
Well that's super annoying, but I guess there's no better way.

Reply:
Actually I think I've found a better way @dpryan79 - commits imminent!